The various git gitweb tests require that the perl HTML::Entities module
be present in order for the tests to work. If it isn't located, then the
tests should be skipped.

This change should be sent upstream.

--- git-2.30.1/t/gitweb-lib.sh.orig	2021-02-25 10:54:09.464770812 +0100
+++ git-2.30.1/t/gitweb-lib.sh	2021-02-25 10:55:36.603973524 +0100
@@ -120,4 +120,9 @@
 	test_done
 }
 
+perl -mHTML::Entities -e 0 >/dev/null 2>&1 || {
+	skip_all='skipping gitweb tests, HTML::Entities module not available'
+	test_done
+}
+
 gitweb_init
